src/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(HEADERS
 add_library(realm_dart SHARED ${SOURCES} ${HEADERS})
 target_compile_definitions(RealmFFIStatic PUBLIC -DRealm_EXPORTS)
 
-target_link_libraries(realm_dart dart-dl RealmFFIStatic Realm::Storage)
+target_link_libraries(realm_dart dart-dl RealmFFIStatic Realm::ObjectStore)
```

Comment:
Curious about this change. What is the purpose?

Reply:
Ah, that's related to the discussion in https://github.com/realm/realm-dart/pull/577#discussion_r875994306.